x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/private_sources_layout.test.tsx

```diff
@@ -13,10 +13,6 @@ import React from 'react';
 
 import { shallow } from 'enzyme';
 
-import { EuiCallOut } from '@elastic/eui';
```

Comment:
Just curious what the reasoning here is? We import the components in other tests, so just curious why we are breaking consistency in this one place.

Reply:
Good catch, I copied an incorrect pattern from [here](https://github.com/elastic/kibana/blob/master/x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/sources_view.test.tsx#L47), but now I realize it was intentional there because the component we're trying to find is not exported. Fixed in commits  https://github.com/elastic/kibana/pull/91555/commits/7a7f0456287d3d0f05072f38daea99b278f4cd04 and https://github.com/elastic/kibana/pull/91555/commits/336bcfb11f8493df03fd77b73d3056b47b4397a1.